pokete.py

```diff
@@ -191,35 +191,35 @@ def playmap_42_npc_21(npc):
                      if i.lvl() >= 50 and i.identifier == "mowcow"]
         if len(poke_list) > 0:
             poke = poke_list[0]
-            npc.text([" < Oh great!", " < You're my saviour!",
+            npc.text([" < Oh great!", " < You're my hero!",
                       f" < You brought me a level {poke.lvl()} Mowcow!",
                       " < I'm thanking you!",
-                      " < Now I can still serve the best MowCow-Burgers!",
+                      " < Now I can still serve the best Mowcow-Burgers!",
```

Comment:
The pascal case MowCow-Burger was actually intentional because it's the restaurants name.

Reply:
My bad, I just went in and replaced every instance of "MowCow" with "Mowcow" to match the name of the Pokete.

Sorry!